module-system/module-implementations/sov-evm/src/tests/genesis_tests.rs

```diff
@@ -73,14 +80,57 @@ fn genesis_cfg_missing_specs() {
 #[test]
 fn genesis_block() {
     let (evm, mut working_set) = get_evm(&TEST_CONFIG);
+    let mut accessory_state = working_set.accessory_state();
 
+    let block_number = evm
+        .block_hashes
+        .get(&GENESIS_HASH, &mut accessory_state)
+        .unwrap();
     let block = evm
         .blocks
-        .get(&0u64, &mut working_set.accessory_state())
+        .get(block_number as usize, &mut accessory_state)
         .unwrap();
 
+    assert_eq!(block_number, 0);
+
     assert_eq!(
         block,
+        SealedBlock {
+            header: SealedHeader {
+                header: Header {
+                    parent_hash: H256::default(),
+                    state_root: KECCAK_EMPTY,
+                    transactions_root: EMPTY_TRANSACTIONS,
+                    receipts_root: EMPTY_RECEIPTS,
+                    logs_bloom: Bloom::default(),
+                    difficulty: U256::ZERO,
+                    number: 0,
+                    gas_limit: ETHEREUM_BLOCK_GAS_LIMIT,
+                    gas_used: 0,
+                    timestamp: 50,
+                    extra_data: Bytes::default(),
+                    mix_hash: H256::default(),
```

Comment:
Is this something for later. If yes can you please add //TODO EVM (so it is easy to grep) to all the places where we use default() temporarily.